draft-tschofenig-rats-psa-token.md

```diff
@@ -338,10 +338,12 @@ This claim MUST be present in a PSA attestation token.
 ### Boot Seed
 {: #sec-boot-seed }
 
-The Boot Seed claim represents a random value created at system boot time that
```

Comment:
why is random removed? if it's 8 bits and not random, then this could be an issue. Also, the PSA SM says that it must be random https://www.psacertified.org/app/uploads/2021/12/JSADEN014_PSA_Certified_SM_V1.1_BET0.pdf

Reply:
The EAT definition also says it must be random https://datatracker.ietf.org/doc/html/draft-ietf-rats-eat-13#section-4.2.14

Reply:
> Also, the PSA SM says that it must be random https://www.psacertified.org/app/uploads/2021/12/JSADEN014_PSA_Certified_SM_V1.1_BET0.pdf

You are right, I've left out a lot of context that you are not aware of.  I should have added it to the PR summary.  Anyway, luckily you asked, so I can give you the background information that should allow you to fill in the blanks.

Rob is currently tweaking the PSA SM wrt boot seed.  The new text is:  _"[...] A boot seed that is unique for each system reset, for example, a random number or a monotonic counter."_.  The text is in the editor copy and it's not been published yet.

Reply:
> The EAT definition also says it must be random https://datatracker.ietf.org/doc/html/draft-ietf-rats-eat-13#section-4.2.14

Good catch!  I think we should ask Laurence to change that to remove the requirement on randomness, which is not necessary.  I think the salient bit is _"value [...] that will allow differentiation of reports from different boot sessions"_.  If one can store that value across resets, they could also use a counter rather than using random sampling.

Reply:
Thanks Thomas. Monontonic counter is a fine approach provided it is large enough. TPMs have been using monotonic boot counters for years, which is like a boot seed, and it has worked well.

Reply:
Filed https://github.com/ietf-rats-wg/eat/issues/187